BLSQ/iaso
# IA-2454 iaso landing page

Providing a default home page for IASO

Related JIRA tickets : IA-2454
## Self proofreading checklist


## Changes

- make a view "home_iaso" available while not connected
- create a new home domain with a default page to display (using custom logo and App label"
- default page displays burger menu or login button if the user is authenticated or not
- allow to override home page in the plugin config
- redirect to the user home url if the user have one in his profile (only on first load)
- add a link to home on the sidebar menu
- add infot tooltip on home page input while editing a profile

## How to test

- make sure your profile doesn't have a home page 
- visit "http://localhost:8081/" while connected, you should see the home page with the sidebar menu
- visit it not connected again, you should see the home page with a login button
- eventually change logo and label of the app with ENV variables:
```
APP_TITLE="Coda 2"
FAVICON_PATH="images/coda/favicon.png"
LOGO_PATH="images/coda/logo.svg"
```
- uncomment line 540 in plugins/polio/js/config.js, revisit "http://localhost:8081/", you should see a single text saying "HOME POLIO"
- add "forms/list" has home page in your profile, revisit "http://localhost:8081/", you should **not** see home page at all and be redirect to the list of forms. Home page should be accessible if you click on the iaso logo in the sidebar menu

## Print screen / video
![Screenshot 2023-09-28 at 10 59 20](https://github.com/BLSQ/iaso/assets/12494624/beac8549-e137-455f-88eb-29ef5cd4ee72)
![Screenshot 2023-09-28 at 10 59 11](https://github.com/BLSQ/iaso/assets/12494624/2fa84b35-e7f6-4cef-a727-a6530eb87487)
![Screenshot 2023-09-28 at 10 58 59](https://github.com/BLSQ/iaso/assets/12494624/0bc62da3-d62d-410a-bc78-08d04efb9e79)


[Whimsical](https://whimsical.com/landing-page-92N39hYvjWnRCHcexwBnWt)
